sqreen/manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+    "integration_id": "sqreen",
+    "name": "sqreen",
+    "manifest_version": "0.0.1",
```

Comment:
please update the version to "1.0.0". Just following convention and it might fail our CI checks.

Reply:
@anshumgargdd for future releases, will we bump it too? Or it'll stay "1.0.0" forever